juego.py

```diff
@@ -163,9 +157,9 @@ def sy(coord_y):
 
     def main(self):
         sonido_menu = load_sound("menu.ogg")
-        jugar = self.fuente_130.render(self.menu_texts[0], True, (0, 0, 255), (0, 0, 0))
-        level = self.fuente_130.render(self.menu_texts[1], True, (0, 0, 255), (0, 0, 0))
-        quit = self.fuente_130.render(self.menu_texts[2], True, (0, 0, 255), (0, 0, 0))
+        jugar = self.fuente_130.render(self.get_translated_text("PLAY"), True, (0, 0, 255), (0, 0, 0))
+        level = self.fuente_130.render(self.get_translated_text("LEVEL"), True, (0, 0, 255), (0, 0, 0))
+        quit = self.fuente_130.render(self.get_translated_text("QUIT"), True, (0, 0, 255), (0, 0, 0))
```

Comment:
Someone with a spanish locale would have these texts not translated as your check in `get_translated_text` only checks in the `en` key when the check should be based on what the locale is, any locale that's not `es` we should use the English text.

Reply:
> check should be based on what the locale is, any locale that's not es we should use the English text

It does exactly that, it finds if the text is a known text in English list, if it is it returns the element at the same index but from user locale's list